### PR TITLE
Fixes on votes

### PR DESCRIPTION
**Pull request by tdesvenain**
fix bugherd #274

### CODE REVIEW

File: assembl/static2/js/app/pages/voteSession.jsx [hunk 1]
```diff
@@ -56,23 +56,24 @@ export type Proposal = {|
 |};
 
 type Props = {
-  loading: boolean,
-  title: string,
-  subTitle: string,
-  seeCurrentVotes: boolean,
+  addGaugeVote: Function,
```
Comment (pascalhuynh) [inline assembl/static2/js/app/pages/voteSession.jsx]: Could you be more specific on the function type ?

File: assembl/static2/js/app/pages/voteSession.jsx [hunk 2]
```diff
@@ -56,23 +56,24 @@ export type Proposal = {|
...
+  addTokenVote: Function,
```
Comment (pascalhuynh) [inline assembl/static2/js/app/pages/voteSession.jsx]: Same here

Comment (tdesvenain) [reply assembl/static2/js/app/pages/voteSession.jsx]: this is not related with the pr, this is just a move for alphabetic order

File: assembl/static2/js/app/pages/voteSession.jsx (hunk 1 above)
Comment (tdesvenain) [reply assembl/static2/js/app/pages/voteSession.jsx]: i improved the return type, this is impossible to improve the parameter types

**Pull request merged**